crypto/fipsmodule/service_indicator/internal.h

```diff
@@ -33,8 +32,9 @@ OPENSSL_INLINE void FIPS_service_indicator_update_state(void) { }
 // key rounds.
 void AES_verify_service_indicator(unsigned key_rounds);
 
-// AEAD APIs work with different parameters. Only internal IV for AES-GCM is
-// meant to be approved.
+// The service indicator checks use different parameters for AEAD APIs than
+// those of other AES modes. AES-GCM is approved only with an internal IV
+// <SP 800-38D  and 8.2>.
```

Comment:
```suggestion
// , see SP 800-38D Sec 8.2.
```
Is Sec 8.2 the correct section?

Reply:
Also, I think we need to specify 8.2.2 not just 8.2, right?

Reply:
Updated